src/components/molecules/navSearch/NavSearchDropdown.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useRef, useState } from "react";
+import { LuChevronsUpDown } from "react-icons/lu";
+
+interface NavSearchDropdownProps {
+  contextualSearchName: string;
+  isEverything: boolean;
+  setIsEverything: (newValue: boolean) => void;
+}
+
+type DropdownOption = {
+  name: string;
+  value: boolean;
+};
+
+export const NavSearchDropdown = ({ contextualSearchName, isEverything, setIsEverything }: NavSearchDropdownProps) => {
+  const ref = useRef(null);
+  const [isOpen, setIsOpen] = useState(false);
+
+  useEffect(() => {
+    const handleFocusOutside = (event: MouseEvent) => {
+      if (ref.current && !ref.current.contains(event.target)) {
+        setIsOpen(false);
+      }
+    };
+    const handleTab = (event: KeyboardEvent) => {
+      if (event.key === "Tab" && ref.current && !ref.current.contains(event.target)) {
+        setIsOpen(false);
+      }
+    };
+
+    document.addEventListener("click", handleFocusOutside, true);
+    document.addEventListener("keyup", handleTab, true);
+
+    return () => {
+      document.removeEventListener("click", handleFocusOutside, true);
+      document.removeEventListener("keyup", handleTab, true);
+    };
+  }, [ref]);
+
+  // Show the other option below the currently selected one
+  const dropdownOptions: DropdownOption[] = [
+    {
+      name: "Everything",
+      value: true,
```

Comment:
✋ suggestion(non-blocking): the code in this component is very clear - but this `value` name keeps throwing me. Would a name like `isEverything` be clearer?

Reply:
`isEverything` is already used as the value prop coming into the component. I've gone with `newIsEverythingValue` instead.